source/Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+ <PropertyGroup>
+   <LangVersion>7.3</LangVersion>
```

Comment:
This sets the version for all projects which don't set their own version.

Reply:
I do remember having syntax errors building against .net 4.0 despite I set the language version to 7.3. 
I guess in the documentation, by saying All .Net Framework uses default version of 7.3, they only included the supported versions of .net framework, which are 4.8 and 4.6(?). .net 4.0 is simply ignored because it was deprecated 8 year ago: https://devblogs.microsoft.com/dotnet/support-ending-for-the-net-framework-4-4-5-and-4-5-1/. It is understandable they don't include this version in "All".

Still, this is a good step forward, at least we have fewer syntax errors! Thank you

Reply:
Ah damn, that's a good point. Yeah It might be "good enough" until we update to latest .net framework version anyway. One day we'll get this solution to .net 6 😅